### PR TITLE
fix the problem of potential NPE in ACL plain

### PR DESCRIPTION
**Pull request by shendongsd**
[#2863]  fix the problem of potential NPE in ACL plain and revise codeStyle

### CODE REVIEW

**Review by panzhi33 — APPROVED** [review]

File: acl/src/main/java/org/apache/rocketmq/acl/plain/PlainPermissionManager.java [hunk 1]
```diff
@@ -50,9 +50,9 @@
 
     private String fileName = System.getProperty("rocketmq.acl.plain.file", DEFAULT_PLAIN_ACL_FILE);
 
-    private  Map<String/** AccessKey **/, PlainAccessResource> plainAccessResourceMap = new HashMap<>();
+    private Map<String/** AccessKey **/, PlainAccessResource> plainAccessResourceMap = new HashMap<>();
```
Comment (vongosling) [inline acl/src/main/java/org/apache/rocketmq/acl/plain/PlainPermissionManager.java:53]: Do you check your style correct, just like here.?https://github.com/apache/rocketmq/pull/2899

File: acl/src/main/java/org/apache/rocketmq/acl/plain/PlainPermissionManager.java [hunk 2]
```diff
@@ -73,14 +73,14 @@ public void load() {
         JSONObject plainAclConfData = AclUtils.getYamlDataObject(fileHome + File.separator + fileName,
             JSONObject.class);
         if (plainAclConfData == null || plainAclConfData.isEmpty()) {
-            throw new AclException(String.format("%s file is not data", fileHome + File.separator + fileName));
+            throw new AclException(String.format("%s file  is not data", fileHome + File.separator + fileName));
```
Comment (vongosling) [inline acl/src/main/java/org/apache/rocketmq/acl/plain/PlainPermissionManager.java:76]: the middle of file and is has two blanks, right?

File: acl/src/main/java/org/apache/rocketmq/acl/plain/PlainPermissionManager.java [hunk 3]
```diff
@@ -133,7 +133,9 @@ public boolean updateAccessConfig(PlainAccessConfig plainAccessConfig) {
 
         Map<String, Object> aclAccessConfigMap = AclUtils.getYamlDataObject(fileHome + File.separator + fileName,
             Map.class);
-
+        if (aclAccessConfigMap == null || aclAccessConfigMap.isEmpty()) {
+            throw new AclException(String.format("%s file  is not data", fileHome + File.separator + fileName));
```
Comment (vongosling) [inline acl/src/main/java/org/apache/rocketmq/acl/plain/PlainPermissionManager.java:137]: what is file is not data, is it right sematic?

File: acl/src/main/java/org/apache/rocketmq/acl/plain/PlainPermissionManager.java (hunk 1 above)
Comment (shendongsd) [reply acl/src/main/java/org/apache/rocketmq/acl/plain/PlainPermissionManager.java:53]: > Do you check your style correct, just like here.?#2899

yes, i checked

File: acl/src/main/java/org/apache/rocketmq/acl/plain/PlainPermissionManager.java [hunk 4]
```diff
@@ -164,20 +166,21 @@ public boolean updateAccessConfig(PlainAccessConfig plainAccessConfig) {
...
     }
 
-    private Map<String, Object> createAclAccessConfigMap(Map<String, Object> existedAccountMap, PlainAccessConfig plainAccessConfig) {
-        
+    private Map<String, Object> createAclAccessConfigMap(Map<String, Object> existedAccoutMap,
+        PlainAccessConfig plainAccessConfig) {
+
         Map<String, Object> newAccountsMap = null;
-        if (existedAccountMap == null) {
+        if (existedAccoutMap == null) {
```
Comment (duhenglucky) [inline acl/src/main/java/org/apache/rocketmq/acl/plain/PlainPermissionManager.java:173]: accout -> account

File: acl/src/main/java/org/apache/rocketmq/acl/plain/PlainPermissionManager.java [hunk 5]
```diff
@@ -133,7 +133,9 @@ public boolean updateAccessConfig(PlainAccessConfig plainAccessConfig) {
 
         Map<String, Object> aclAccessConfigMap = AclUtils.getYamlDataObject(fileHome + File.separator + fileName,
             Map.class);
-
+        if (aclAccessConfigMap == null || aclAccessConfigMap.isEmpty()) {
+            throw new AclException(String.format("%s file not found or empty", fileHome + File.separator + fileName));
```
Comment (ShannonDing) [inline acl/src/main/java/org/apache/rocketmq/acl/plain/PlainPermissionManager.java:137]: "the xxx file  **is** not found or empty " seems better.

**Review by duhenglucky — APPROVED** [review]

**Pull request merged**